.circleci/config.yml

```diff
@@ -167,21 +167,18 @@ workflows:
             - unit_test
             - integration_test
             - build_docker
-
-#      - request-preprod-approval:
-#          type: approval
-#          requires:
-#            - deploy_dev
-#      - hmpps/deploy_env:
-#          name: deploy_preprod
-#          env: "preprod"
-#          context:
-#            - hmpps-common-vars
-#            - hmpps-template-typescript-preprod
-#          requires:
-#            - request-preprod-approval
-#          requires:
-#            - deploy_preprod
+      - request-preprod-approval:
+          type: approval
+          requires:
+            - deploy_dev
+      - hmpps/deploy_env:
+          name: deploy_preprod
+          env: "preprod"
+          context:
+            - hmpps-common-vars
+            - hmpps-template-typescript-preprod
```

Comment:
Sorry John, I merge your PR but realised that we might have a mistake on that. I think we should change line 179 by `court-probation-preprod`, don't you think too?

Reply:
Yep, good spot!